templates/blog/product-cards/_opensearch.html

```diff
@@ -1,18 +1,16 @@
 <div class="p-card js-product-card u-hide">
   <img class="p-card__thumbnail"
-       src="https://assets.ubuntu.com/v1/d6fc5627-Canonical%20Charmed%20OpenSearch%20logo.svg"
+       src="https://assets.ubuntu.com/v1/f8dde6f0-opensearch-logo.png"
```

Comment:
similar here, the image is almost invisible, it's too small. Is that okay with the PM?